openapi-module/src/generators/curl-generator.js

```diff
@@ -0,0 +1,73 @@
+const fs = require('fs');
+const { parseOpenAPIDocument } = require('../parsers/openapi-parser');
+const { generateFakeField, generateFakeData } = require('./field-mapping');
+const querystring = require('querystring');
+
+async function generateCurlCommands(openApiFilePath, outputFilePath) {
+    const endpoints = await parseOpenAPIDocument(openApiFilePath);
+
+    if (!endpoints) {
+        console.error('Failed to parse OpenAPI document.');
+        return;
+    }
+
+    const commands = (await Promise.all(endpoints.map(endpoint => createCurlCommand(endpoint)))).join('\n\n');
+    fs.writeFileSync(outputFilePath, commands);
+    console.log(`cURL commands generated and saved to ${outputFilePath}`);
+}
+
+async function createCurlCommand(endpoint) {
+    let url = `http://localhost:3000${endpoint.path.replace(/{(.*?)}/g, (_, key) => `\${${key}}`)}`;
```

Comment:
Please remove all the hardcoded 3000 ports and use a static global constant = 4000.